src/server/protocol.ts

```diff
@@ -83,6 +83,9 @@ namespace ts.server.protocol {
         SignatureHelp = "signatureHelp",
         /* @internal */
         SignatureHelpFull = "signatureHelp-full",
+        SourceDefinitionAndBoundSpan = "sourceDefinitionAndBoundSpan",

```

Comment:
I think we may already have discussed this but, unfortunately, I've forgotten.  Why wouldn't this be modelled on `definition[-full]`?  I believe the primary (only?) difference is that `definitionAndBoundSpan` includes the span for underlining, which this command seems unlikely to require, given the expected perf.